### PR TITLE
Remove outdated texts from previous courses

### PR DESCRIPTION
**Pull request by AMashoshyna**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -5,25 +5,21 @@
...
-  + A: Offline course started 26.01.2019. P2P course started 27.01.2019.
+  + A: Preparatory stage of Frontend Course starts 12.10.20. Date of main course to be announced.
 - Q: Is this course available online?
-  + A: Peer-to-peer course is available online. This means, that we will provide tasks, recomendations for self-education, for code-review, acceptance criteria, etc. Also, we will create tasks from offline part - we might put up some of the lecture recordings, so you could try working on this at your own pace and ask questions in the [chat]
+  + A: Preparatory stage of the course includes free learning materials. As for main course, we plan to make records of all lectures & workshops and make them available online, but live presence is for course students only.
 - Q: Where are courses going to take place?
-  + A: Offline course will take place at [Cogniance office](https://goo.gl/maps/cVxLSCpWPUB2). Peer-to-peer course is mostly online.
+  + A: All lectures & workshops will be held online.
 - Q: At what time are lectures gonna be?
   + A: From 11:00 - 12:00 to 17:00 - 18:00. One day on the weekend - Saturdays.
```
Comment (Croftyland) [inline faq.md:16]: This question is supposed to be also modified

Comment (AMashoshyna) [reply faq.md]: Removed it for now

File: SCHEDULE.md [hunk 2]
```diff
@@ -1,29 +0,0 @@
-# Frontend Course 2019
```
Comment (A-Ostrovnyy) [inline SCHEDULE.md:1]: Maybe no need to delete this file, just remove all content except an empty schedule table template?
After the first lecture, we will create this file again)

Comment (AMashoshyna) [reply SCHEDULE.md:1]: Good point! added the file back.

**Comment by zonzujiro** [comment]
Probably this also needs to be changed https://github.com/kottans/kottans.github.io/blob/source/html-dev/home/4-faq.html#L35

**Review by zonzujiro — APPROVED** [review]

**Review by Croftyland — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -5,15 +5,17 @@
...
 You can use these materials for self-education as well.
 
+### Dates & Deadlines
+
+For 2020-2021 course: preparatory stage starts 12.10.2020 and ends in December. Main course starts in December (exact dates to be announced)
 
 ### Admission
 
-TODO
+To join main course track, the applicants are required to finish tasks listed in [Stage 0. Self-Study](contents.md#stage-0-self-study) part of the course and to pass successfully the interview with mentors. We highly encourage you to be active in your study process and help other students: answer questions in chants, review pull-requests, report possible errors and suggest solutions. Don’t hesitate to ask questions and let mentors know if you are stuck and need help. Your progress is in your hands.
```
Comment (Croftyland) [inline README.md:18]: We should mention that is one of our criteria for passing to the course

**Pull request merged**